capstone-frontend/src/components/MyClubs.jsx

```diff
@@ -67,14 +73,14 @@ class MyClubs extends Component {
         <Row>
           <Col xs={12} className="title"> My Clubs </Col>
         </Row>
-          <Link to="/createclub"> 
-            <Button variant="primary">
-              Create New Club 
+        <Link to="/createclub">
+          <Button variant="primary">
```

Comment:
nit: Can you fix the alignment of the Button opening and closing tags?

Reply:
Done.